test/unit/test_hpe3par_snapshot.py

```diff
@@ -150,7 +163,10 @@ def test_main_exit_present(self, mock_create_snapshot, mock_module, mock_client)
             'allow_remote_copy_parent': None,
             'new_name': None,
             'rm_exp_time': None,
-            'state': 'present'
+            'state': 'present',
```

Comment:
This is not clear to me? what exactly are we trying here?

Reply:
'present' state means creating snapshot.